docs/developing/dev_environment/local_proxy_remote_solana.md

```diff
@@ -85,14 +86,15 @@ export SOLANA_URL=http://api.devnet.solana.com/
 export REVISION=latest
 ```
 
-3. Download the `docker-compose` [file](https://github.com/neonlabsorg/proxy-model.py/blob/develop/docker-compose/docker-compose-remote-solana.yml). This file should be placed in the same folder with the `keys/` directory.
+3. Download the `docker-compose` [file](https://github.com/neonlabsorg/neon-proxy.py/blob/b3bd1a298f3a437cb48379f348ed71268af382cc/docker-compose.yml). This file should be placed in the same folder with the `keys/` directory.
 ```bash
-wget https://raw.githubusercontent.com/neonlabsorg/proxy-model.py/develop/docker-compose/docker-compose-remote-solana.yml
+wget https://raw.githubusercontent.com/neonlabsorg/neon-proxy.py/b3bd1a298f3a437cb48379f348ed71268af382cc/docker-compose.yml
 ```
 
 4. Start the local environment.
 ```bash   
 docker-compose -f docker-compose-remote-solana.yml up postgres dbcreation proxy indexer -d
+SOLANA_URL=[working solana url] SOLANA_KEY_FOR_EVM_CONFIG=[public key of id.json] docker-compose up -d
```

Comment:
if you defined env above you don not need to place it before compose run command

Reply:
Plase remove the section below
````
The console output should look like this:

Creating postgres ... done
Creating dbcreation ... done
Creating indexer ... done
Creating proxy ... done
````

If you want to add smth about container statuses you can write "The containers have to be started successfully" right after the run command.

Reply:
But, how the user identify that conainers are started with no troubles?

Reply:
eth developers are able to read output and validate was it successful or not

Reply:
Updated with adding step 5. Check the local environment.